iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/pipeconsensus/PipeConsensusAsyncConnector.java

```diff
@@ -521,4 +551,25 @@ public int getTransferBufferSize() {
   public int getRetryBufferSize() {
     return retryEventQueue.size();
   }
+
+  @Override
+  public long getLocalUserWriteProgress() {
+    PipeStaticMeta staticMeta = PipeAgent.task().getPipeStaticMeta(consensusPipeName);
```

Comment:
Better not expose the "staticMeta" in PipeDataNodeTaskAgent only... There may be potential altering to it. It's better to add a "getCreationTime" in it.

Reply:
fixed